### DISCUSSION TITLE
Docker How to configure doris to start the observer service using docker-compose?

### DISCUSSION THREAD

**Discussion by mpv945** [apache/doris#38702] [A - General / Q&A]
I want to build 5 FE nodes, 3 Followers and 2 Observers. 8 BE nodes and need to run Broker. How do I configure my docker-compose.yml?

**Comment by ixzc** [comment]
docker-compose doesn't support observer. 
3 followers are enough.

**Reply by mpv945** [reply]
Does docker run -d support observer starting?
